pkg/cli/add_vcluster_helm.go

```diff
@@ -33,22 +39,105 @@ func AddVClusterHelm(
 	vClusterName string,
 	log log.Logger,
 ) error {
-	// check if vCluster exists
-	vCluster, err := find.GetVCluster(ctx, globalFlags.Context, vClusterName, globalFlags.Namespace, log)
-	if err != nil {
-		return err
+	var vClusters []find.VCluster
+	if options.All {
+		log.Debugf("add vcluster called with --all flag")
+		kubeClientConfig := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+			clientcmd.NewDefaultClientConfigLoadingRules(), &clientcmd.ConfigOverrides{
+				CurrentContext: globalFlags.Context,
+			})
+		hostClusterRestConfig, err := kubeClientConfig.ClientConfig()
+		if err != nil {
+			return err
+		}
+		hostKubeClient, err := kubernetes.NewForConfig(hostClusterRestConfig)
+		if err != nil {
+			return err
+		}
+		namespaces, err := hostKubeClient.CoreV1().Namespaces().List(ctx, metav1.ListOptions{})
+		if err != nil {
+			return err
+		}
+		log.Debugf("looking for vclusters in %d namespaces", len(namespaces.Items))
+		for _, ns := range namespaces.Items {
+			log.Infof("looking for a vcluster in %s namespace", ns.GetName())
```

Comment:
```suggestion
			log.Infof("looking for a vclusters in %s namespace", ns.GetName())
```

Reply:
because of the trademark, we should either use, `virtual cluster` or `vcluster instance` but not use `vcluster(s)` like that